modules/placekey/redshift/test/integration/test_ISVALID.py

```diff
@@ -0,0 +1,46 @@
+from test_utils import run_query
+
+
+def test_isvalid():
+
+    result = run_query('''
+        SELECT @@RS_PREFIX@@placekey.ISVALID(NULL)
+        UNION ALL
+        SELECT @@RS_PREFIX@@placekey.ISVALID(\'@abc\')
+        UNION ALL
+        SELECT @@RS_PREFIX@@placekey.ISVALID(\'abc-xyz\')
+        UNION ALL
+        SELECT @@RS_PREFIX@@placekey.ISVALID(\'abcxyz234\')
+        UNION ALL
+        SELECT @@RS_PREFIX@@placekey.ISVALID(\'abc@abc-234-xyz\')
+        UNION ALL
+        SELECT @@RS_PREFIX@@placekey.ISVALID(\'ebc-345@abc-234-xyz\')
+        UNION ALL
+        SELECT @@RS_PREFIX@@placekey.ISVALID(\'bcd-345@\')
+        UNION ALL
+        SELECT @@RS_PREFIX@@placekey.ISVALID(\'22-zzz@abc-234-xyz\')
+        UNION ALL
+        SELECT @@RS_PREFIX@@placekey.ISVALID(\'abc-234-xyz\')
+        UNION ALL
+        SELECT @@RS_PREFIX@@placekey.ISVALID(\'@abc-234-xyz\')
+        UNION ALL
+        SELECT @@RS_PREFIX@@placekey.ISVALID(\'bcd-2u4-xez\')
+        UNION ALL
+        SELECT @@RS_PREFIX@@placekey.ISVALID(\'zzz@abc-234-xyz\')
+        UNION ALL
+        SELECT @@RS_PREFIX@@placekey.ISVALID(\'222-zzz@abc-234-xyz\')
```

Comment:
Not sure that you need the scape character \'. Same in other tests

Reply:
Yes, you are right. I started not using the ''' ''' quotes and in that case the scape character was needed. Removing them!